libs/application-generic/src/services/analytic-logs/workflow-run/workflow-run.repository.ts

```diff
@@ -453,4 +453,263 @@ export class WorkflowRunRepository extends LogRepository<typeof workflowRunSchem
 
     return template.name.toLowerCase().replace(/\s+/g, '_');
   }
+
+  async getWorkflowVolumeData(
+    environmentId: string,
+    organizationId: string,
+    startDate: Date,
+    endDate: Date
+  ): Promise<Array<{ workflow_name: string; count: string }>> {
+    const query = `
+      SELECT 
+        workflow_name,
+        count(*) as count
+      FROM workflow_runs FINAL
+      WHERE 
+        environment_id = {environmentId:String} 
+        AND organization_id = {organizationId:String}
+        AND created_at >= {startDate:DateTime64(3)}
+        AND created_at <= {endDate:DateTime64(3)}
+        AND status = 'success'
```

Comment:
Are we interested only in the "success" workflow runs? what if the user has a lot of pending workflows? (in-progress or delayed/digested)

Reply:
Yeah, that's valid